site_source_files/content/blog/2021/selenium-4-observability.md

```diff
@@ -0,0 +1,64 @@
++++
+Description = "Observability in Selenium 4"
+Title = "Observability in Selenium 4"
+Date = 2021-04-26
+Author = "Puja Jagani"
+AuthorLink = "https://twitter.com/pujagani"
+tags = ["selenium", "grid"]
+categories = ["general"]
++++
+
+Selenium server is now power-packed with [Opentelemetry](https://opentelemetry.io/) to help troubleshoot issues and optimize performance. Thus, making the Selenium server observable! 
```

Comment:
Starting with "why should I care?" is a nice way to hook people into an article. Perhaps something like...

```
Diagnosing problems when the Selenium server isn't working has never been easy. With Selenium 4, we've added support for [OpenTelemetry](https://opentelemetry.io/) to help you troubleshoot issues and optimize performance. By making the Selenium server observable, we're putting more power into your hands.
```

… or something similar?